README.md

```diff
@@ -5,23 +5,20 @@ This repository contains a reference AWS Platform Configuration for
 internal cloud platforms with AWS and offering a self-service API to your internal
 development teams.
 
-This platform provides APIs to provision fully configured EKS clusters, with
-secure networking, and stateful cloud services (RDS) designed to securely
-connect to the nodes in each EKS cluster — all composed using cloud service
-primitives from the [Official Upbound AWS
-Provider](https://marketplace.upbound.io/providers/upbound/provider-aws). App
-deployments can securely connect to the infrastructure they need using secrets
-distributed directly to the app namespace.
+This platform offers APIs for setting up fully configured EKS clusters with secure networking, stateful cloud services (RDS) that can securely connect to the EKS clusters, an Observability Stack, and a GitOps System. All these components are built using cloud service tools from the [Official Upbound AWS
+Provider](https://marketplace.upbound.io/providers/upbound/provider-aws). App deployments can securely access the necessary infrastructure through secrets distributed directly to the app namespace.
```

Comment:
Nit: can you please wrap the text to a standard original 80-char length?

Reply:
done ;)